recipes/wpa-supplicant/wpa-supplicant/wpa_supplicant-2.5-fix-libressl.patch

```diff
@@ -0,0 +1,65 @@
+--- wpa_supplicant-2.5/src/crypto/tls_openssl.c.orig	2016-01-23 14:53:34.479639958 +0000
```

Comment:
Please add a comment on where you found this patch, why it is needed and the upstream status (should it be reported?)...